discovery/pkg/apigee/provision.go

```diff
@@ -157,18 +167,23 @@ func (p provisioner) ApplicationRequestProvision(req prov.ApplicationRequest) pr
 		return failed(ps, fmt.Errorf("failed to create app: %s", err))
 	}
 
+	log.Infof("provisioned app %s", req.GetManagedApplicationName())
+
 	return ps.Success()
 }
 
 // CredentialDeprovision - Return success because there are no credentials to remove until the app is deleted
 func (p provisioner) CredentialDeprovision(_ prov.CredentialRequest) prov.RequestStatus {
+	msg := "credentials will be removed when the azure subscription is deleted"
```

Comment:
message should be apigee instead of azure

Reply:
@tsjohns9 ^^^ this one. Then it's good

Reply:
fixed